### ISSUE TITLE
Create two more metrics for alertness level

### ISSUE THREAD

**Issue by atquail** [redarrowlabs/tournavation-2015#54]
Users should be able to select one alertness level from a selection of 5.  Right now, the application only allows one selection from a list of three.

We need to define the two new metrics (will update this issue when known).

Essentially, create two more metrics (and tiles) for alertness level.  Participants can then select one alertness level from a list of 5.

**Comment by kszatmary** [comment]
Here are some words for consideration.  OK is not very descriptive, so maybe one of these could jazz it up. 
On the low alert side:
Drowsy 
Need a Nap
Hungry or Thirsty
Slow

On the ok side:
Refreshed
Active
Energized
Focused
steady and ready
Sharp
Ready for Anything
Ready to Learn
Well-Rested

5 Possible metrics
Drowsy
Slow-Moving
Well-Rested or Focused
Distracted
Wired or Hyper

**Comment by gentilim** [comment]
I agree.  I think "invoking play" and using more "relate-able" terms is a great idea!  Would be great to find works that resonate well with our target audience.  What are the kids saying these days?

**Comment by kszatmary** [comment]
Suggest capitalizing the letter A for Asleep to be consistent with all other labels.
